General Iteration (#322)
Roblox/creator-docs: content/en-us/tutorials/fundamentals/coding-5/intro-to-dictionaries.md

> Many scripts will often need to go through dictionaries or arrays. But sometimes, you may not know the start and end of these data structures. For example, a dictionary of players may be changing, or players may have an array inventory of varying sizes.

Reviewer: ```suggestion
Many scripts need to go through dictionaries or arrays, but you may not always know the start and end of these data structures. For example, a dictionary of players may be changing, or players may have an array inventory of varying sizes.
```

> In these situations, you can use `pairs()` and `ipairs()`. These can be used with a for loop to go through each element without known start and end points.

Reviewer: ```suggestion
In these situations, you can use `pairs()` and `ipairs()` in a `for` loop to go through each element without known start and end points.
```

> In these situations, you can use `pairs()` and `ipairs()`. These can be used with a for loop to go through each element without known start and end points.
> 
> It is also important to note that you can use [general iteration](https://luau-lang.org/performance#optimized-table-iteration) to loop through elements without the need for `pairs()` or `ipairs()`.

Reviewer: ```suggestion
In these situations, you can use `pairs()` and `ipairs()` in a `for` loop to go through each element without known start and end points. You can also use general iteration through the `in` keyword to loop through elements without the need for `pairs()` or `ipairs()`.
```
